bases/sfmc-app/@{{ cookiecutter.project_slug }}/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "@{{ service_name }}",
+  "name": "@{{ cookiecutter.service_name }} }}",
```

are the double closing bracket typos or needed?
```suggestion
  "name": "@{{ cookiecutter.service_name }}",
```